PointerStar/Client/Pages/Room.razor

```diff
@@ -259,4 +272,18 @@
             await ViewModel.SubmitDialogAsync();
         }
     }
+
+    
+    private async Task OnClickClipboard(object sender)
+    {
+        CopyButtonText = "Copied";
```

Comment:
I would wait to change the text/icon until after calling the service.

Reply:
it works right visually but conceptually feels wrong, because I am not awaiting my call, I changed the method to return a ValueTask instead of a Task, I tried to capture the result as a ValueTask and check for the IsCompleted value of the result but it didn't work